info_preprocess_header.h

```diff
@@ -1,3 +1,3 @@
 // Auto-generated
 
-#define BUILD_NUMBER 71
+#define BUILD_NUMBER 72
```

Comment:
Nepřijde ti nepraktický, jak se  ti v každým commitu udělá změna v build number? 😏 Nenapadá tě na to nějaký řešení? 🤔

Reply:
Hodil jsem info_preprocess_header.h do gitignoru. Jestli jsi myslel tohle. Jinak dále nepraktické mi přijde, že se to jmenuje BUILD_NUMBER a ne COMMIT_NUMBER. 😁

Reply:
.gitignore je správná odpověď ☝️😄